src/systemcmds/tests/test_dataman.c

```diff
@@ -63,6 +63,16 @@ int test_dataman(int argc, char *argv[]);
 
 #define NUM_MISSIONS_TEST 50
 
+typedef union dataman_max_size_t {
+	struct home_position_s		safe_item;
+	struct mission_item_s		mission_item;
+	struct mission_s			mission;
+	struct fence_vertex_s		vertex;
+	struct dataman_compat_s		compat;
+} dataman_max_size_t;
+
+#define DM_MAX_DATA_SIZE sizeof(dataman_max_size_t)
```

Comment:
I think we can simplify this to `#define DM_MAX_DATA_SIZE sizeof(struct mission_s)` so we don't need the struct.

Also, not related, but I noticed the return of `task_main` is not checked, thus the unit test can fail but the result still shows as success.